### DISCUSSION TITLE
SEI parameter info

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3118] [Q&A] [answered]
I am working on a custom parameter set and I want to know, what is being referred to as "Ratio of lithium moles to SEI moles"?

**Comment by brosaplanella** [comment, marked as answer]
It refers to the ratio of stoichiometry coefficients between lithium and SEI in the SEI growth reaction.

For example, in the [Yang et al (2017) paper](https://www.sciencedirect.com/science/article/pii/S0378775317307619) it would be 2, as from equation (6b) we see that in the reaction there are 2 mol of Li to form 1 mol of SEI.

**Reply by brosaplanella** [reply]
Note that this parameter is only needed if you run SEI models, otherwise it is not used.
